### ISSUE TITLE
Add wikitionary

### ISSUE THREAD

**Issue by jokerkeny** [crimx/ext-saladict#345]
- 类型：建议
既然已经词典选项里已经有Wikipedia了，建议添加**wikitionary**，比wikipedia更适合查询词本身的信息，如词源、词义、读音、及各种语言的意思。

**Comment by crimx** [comment]
你好，

> 建议添加wikitionary，比wikipedia更适合查询词本身的信息

这话没错的，这里只是想指明加入 Wikipedia 是为了用 Wikipedia 的功能（百科）而不是用它不擅长的功能（词典）。

> 如词源、词义、读音、及各种语言的意思

Wikitionary 最终会加上去的，现在以上几点 Saladict 都提供了专门的词典，所以不是很紧急。

**Comment by Sagitee** [comment]
Thank you for your really great addon.

Wiktionary has a very different, multi-language approach compared with other online dictionaries and at the same time is today one of the most complete and authoritative ones, even compared with language specific dictionaries like Cambridge’s or Longman’s.
So, I believe that the addition of Wiktionary (mostly its more complete, English one, but if possible, the others too) to Saladict will multiply the usefulness of both.

And by the way, a question:
How much feasible-possible might be a Saladict option for the custom dictionaries addition by the users, with the use of some basic parameters, like some other, mainly searching addons do? (like https://addons.mozilla.org/en-GB/firefox/addon/contextsearch-web-ext/ or https://addons.mozilla.org/en-GB/firefox/addon/contextual-search/)

![191030_115120_capture_001](https://user-images.githubusercontent.com/50146787/67850240-ad79cd00-fb10-11e9-9382-80ebd676a376.jpg)

![191030_115347_capture_002](https://user-images.githubusercontent.com/50146787/67850250-b66a9e80-fb10-11e9-9883-af96c1b57e77.jpg)

**Comment by crimx** [comment]
@Sagitee You are absolutely right. But I think the current combination is sufficient for the majority of users. The payoff of adding yet another dictionary with a bit extra authority is not motivating enough for me to get my hands dirty.

As for the context menus, Saladict already supports adding custom links to context menus.

**Comment by Sagitee** [comment]
> You are absolutely right. But I think the current combination is sufficient for the majority of users. The payoff of adding yet another dictionary with a bit extra authority is not motivating enough for me to get my hands dirty.

You are absolutely right too.

>As for the context menus, Saladict already supports adding custom links to context menus.

I had already move this question as a separate issue. Please, see #545.

**Comment by eromoe** [comment]
比如像 s.t.  这类缩写只有 wiktionary 上有，其他引擎搜出来的都非常奇怪

**Comment by funnybones2015** [comment]
顶起来，希望能早日支持Wiktionary！！！
